backend/dataall/core/permissions/services/permission_service.py

```diff
@@ -89,3 +90,10 @@ def init_permissions(session: Session) -> [str]:
             TENANT_ALL_WITH_DESC,
             PermissionType.TENANT,
         )
+
```

Comment:
I think we should move this to the organizations service as it is only relevant for organization permissions. The same way we also check environment permissions in `backend/dataall/core/environment/services/environment_service.py:534`. 

It is also good to validate that the permission exists in the db when listing

Reply:
It's still permissions, not really the organization thing. 
I would prefer to keep it here, if you are not strongly against it.

Reply:
There are 2 arguments to keep organization permissions in organizations. The first one is to avoid circular dependencies. Right now it does not matter because core is always imported, but if in the future we want to replace the permissions module by another module, it should be an standalone module; organizations, environments depend on permissions. 

The second reason is that for environments we check the permissions in environment. As a developer if I see org permissions in permissions i would assume "all permissions are defined here" and probably miss that environment permissions are defined in environment.

What is more important is the check that it is done in environments. That's what Imeant by "It is also good to validate that the permission exists in the db when listing"

```
    @staticmethod
    def list_group_invitation_permissions():
        with get_context().db_engine.scoped_session() as session:
            group_invitation_permissions = []
            for p in environment_permissions.ENVIRONMENT_INVITATION_REQUEST:
                group_invitation_permissions.append(
                    PermissionRepository.find_permission_by_name(
                        session=session,
                        permission_name=p,
                        permission_type=PermissionType.RESOURCE.name,
                    )
                )
            return group_invitation_permissions
```

Reply:
Ok, you persuade me.